src/components/core/navbar/index.tsx

```diff
@@ -18,7 +18,169 @@ import { getPosts } from "@/redux/reducers/postsReducer";
 import { saveBookmarkToStore } from "@/redux/reducers/bookmarkReducer";
 import { parseCookies } from "nookies";
 
+import Shepherd from 'shepherd.js'; 
+import '../../../../node_modules/shepherd.js/dist/cjs/css/shepherd.css';
+import './styles.css';
+
+
+const startTour = () => {
```

Comment:
It would be better if you create a separate file in utils for all the shepherd configuration